config/routes.rb

```diff
@@ -21,4 +21,6 @@
       end
     end
   end
+
+  match '*path', via: :all, to: 'api/v0/base#error_404'
```

Comment:
seems a little weird for a catchall route to map to "api v0"?

Reply:
truth.  putting it back the way i had it originally, in application controller. thanks!